pyproject.toml

```diff
@@ -49,3 +53,81 @@ test = [
 [tool.hatch]
   build.hooks.vcs.version-file = "src/array_api_typing/_version.py"
   version.source               = "vcs"
+
+
+[tool.coverage]
+  report.exclude_also = ['\.\.\.', 'if typing.TYPE_CHECKING:']
+  run.source          = ["array-api-typing"]
+  run.branch          = true
+
+
+[tool.mypy]
+  files                    = ["src", "tests"]
+  python_version           = "3.10"
+
+  strict                   = true
+  disallow_incomplete_defs = true
+  disallow_untyped_defs    = true
+  disable_bytearray_promotion = true  # Note(2024-12-05): these are private flags
+  disable_memoryview_promotion = true  # Note(2024-12-05): these are private flags
+  enable_error_code        = ["ignore-without-code", "redundant-expr", "truthy-bool"]
+
+  warn_return_any          = true
+  warn_unreachable         = true
+  warn_unused_configs      = true
```

Comment:
inconsistent alignment

maybe it's easiest to no align the values 🤷🏻

Reply:
Agreed. Done.